packages/relay/src/lib/precheck.ts

```diff
@@ -376,4 +380,17 @@ export class Precheck {
       throw predefined.UNSUPPORTED_TRANSACTION_TYPE;
     }
   }
+
+  /**
+   * Checks if the receiver account exists.
+   * @param {Transaction} tx - The transaction.
+   * @param {RequestDetails} requestDetails - The request details for logging and tracking.
+   */
+  async receiverAccount(tx: Transaction, requestDetails: RequestDetails) {
+    const verifyAccount = await this.mirrorNodeClient.getAccount(tx.to!, requestDetails);
+
+    if (verifyAccount !== null && verifyAccount.receiver_sig_required === true) {
```

Comment:
Could you kindly did some extra research and add a comment here stating why we want to make sure `receiver_sig_required` attribute is turned off?